nat_conntracker/flow_parser.py

```diff
@@ -11,7 +11,8 @@ def __init__(self, conntracker, logger):
         self._conntracker = conntracker
         self._logger = logger
 
-    def handle_events(self, stream):
+    def handle_events(self, stream, is_done=None):
+        is_done = is_done if is_done is not None else lambda: False
```

Comment:
I'm not sure I understand this line. What's the purpose of assigning `lambda: False` to `is_done`? Why not:
`is_done = is_done or False`
or just:
`is_done=False` as the keyword argument?